package.json

```diff
@@ -74,6 +74,11 @@
                     "default": "latest",
                     "markdownDescription": "Initial offset to read from for new consumers. See [here](command:vscode-kafka.open.docs.page?%5B%7B%22page%22%3A%22Consuming%22%2C%22section%22%3A%22kafkaconsumersoffset%22%7D%5D) for more information."
                 },
+                "kafka.consumers.headers.print" : {
+                    "type": "boolean",
+                    "default": "false",
+                    "markdownDescription": "Print headers of message"
```

Comment:
Add a link to your doc with `See [here](command:vscode-kafka.open.docs.page?...`

Reply:
Link added